src/core/context.py

```diff
@@ -36,6 +43,13 @@ def __init__(self, model: Model, config: ConfigData):
             extra_user_roles="superuser",
         )
 
+        # FIXME: The database_name currently requested is a dummy name
+        # This should be replaced with the name of actual znode when znode created
+        # by zookeeper charm has enough permissions for Kyuubi to work
+        self.zookeeper_requirer_data = DatabaseRequirerData(
+            self.model, ZOOKEEPER_REL, database_name=HA_ZNODE_NAME_TEMP
```

Comment:
As a workaround, we ask Zookeeper to just create a dummy node that we won't use (because this znode does not have enough ACL and permissions for HA to work).